scripts/test_hwprobe.sh

```diff
@@ -14,13 +14,12 @@
 #
 #    ./test_hwprobe.sh /path/to/socket-count-test-data/dumps/
 #
-
-SYS_DUMPS_PATH="$1"
+SYS_DUMPS_PATH="${1:-$HOME/src/socket-count-test-data/dumps}"
```

Comment:
Is this a directory I need to create on my machine? When I try to run the script it errors with 

find: ‘/home/wpoteat/src/socket-count-test-data/dumps’: No such file or directory